likeminds-chat-reactnative-integration/ChatSX/constants/Styles.ts

```diff
@@ -78,6 +77,9 @@ export class STYLES {
     "dark-content": "dark-content",
     "light-content": "light-content",
   };
+  static $CHAT_BUBBLE_STYLE: ChatBubbleStyles = {};
+  static $INPUT_BOX_STYLE: InputBoxStyles = {};
+  static $REACTION_LIST_STYLE: ReactionListStyles = {};
```

Comment:
the variables are empty object, so when you are using these variables. Does the `chatBubbleStyle ? chatBubbleStyle : null` check works